packages/buendia-site-test-integration/data/usr/share/buendia/tests/60-site-test-integration-restore

```diff
@@ -0,0 +1,50 @@
+test_10_create_patient () {
+    USER_ID=$(head -c8 /dev/random | base64)
+    openmrs_post patients/ >create-patient.json <<END
+        {"id": "$USER_ID", "given_name": "Louis", "family_name": "Pasteur", "sex": "M", "birthdate": "1924-08-20"}
+END
+    cat create-patient.json
+    grep '"uuid":"........-....-....-....-............"' create-patient.json
+}
+
+test_20_get_patient_list () {
+    openmrs_get patients/ >patients-before.json
+    grep '"uuid":"........-....-....-....-............"' patients-before.json
+}
+
+test_30_create_backup () {
+    mount_loopback
+    # TODO: this only fails because it can't mount /dev/loop[1-7]... which
+    # shouldn't count as a failure
+    execute_cron_right_now backup || true
+    [ -d loop/backup.$(date +%Y-%m-%d) ]
+}
+
+test_40_delete_patients () {
+    echo "DELETE FROM encounter; DELETE FROM patient;" | execute_openmrs_sql
+}
+
+test_50_confirm_empty_patient_list () {
+    openmrs_get patients/ >patients-during.json
+    if diff patients-before.json patients-during.json; then
+        echo
+        echo "patient list is unchanged after deletion!"
+        echo
+        return 1
+    fi
+    return 0
+}
+
+test_60_restore_from_backup () {
+    mv loop/backup.$(date +%Y-%m-%d) loop/restore || return 1
+    ( 
+        flock --verbose -w 30 -n 9 || exit 1
+        flock -u -n 9
+        execute_cron_right_now backup
+    ) 9>/var/run/lock/buendia-restore
```

Comment:
(The `flock -u` is there because `buendia-restore` will need to lock the same file itself, when it starts up deep inside a set of nested child processes)

Reply:
I solved this problem by moving the crontab out of the way. This way I could get rid of the weird file locking code